Marlin/src/module/temperature.cpp

```diff
@@ -870,6 +873,7 @@ volatile bool Temperature::raw_temps_ready = false;
 
     // move to center of bed, just above bed height and cool with max fan
     SERIAL_ECHOLNPGM("Moving to tuning position");
+    LCD_MESSAGE_F("Moving to tuning pos");
```

Comment:
I’d recommend these get called from the language file(s) so they can be translated. Also, longer messages will scroll with  `STATUS_MESSAGE_SCROLLING`.

Reply:
Thank you @thisiskeithb. I need guidance here.

Do you mean I should define something like `MSG_MOVING_TO_TUNING_POS` and do the equivalent of `LCD_MESSAGE(MSG_HEATING)`? Or something else?

What is the normal procedure when you have a new string but no translations?

And are you suggesting I do something with `STATUS_MESSAGE_SCROLLING` or just suggesting that I do not truncate my messages?

Reply:
Thanks for the help. Now using language files.

Reply:
Sorry for not being clearer on Discord. Serial strings (`SERIAL_ECHOLNPGM()`, etc.) *should not* be translated and should be in English.